config/software/berkshelf.rb

```diff
@@ -43,4 +43,18 @@
 
   gem "install pkg/berkshelf-*.gem" \
       " --no-ri --no-rdoc", env: env
+
+  if windows?
+    block "Clean up large object files" do
+      # find the embedded rubygems dir and clean it up for globbing
+      gem_dir = "#{install_dir}/embedded/lib/ruby/gems/*/gems".gsub(/\\/, '/')
+
+      # find all the static *.a files in the dep-selector-libgecode gem(s)
+      # we don't use and delete them
+      Dir.glob("#{gem_dir}/dep-selector-libgecode*/**/*.a").each do |f|
```

Comment:
also its a bit odd that berkshelf is cleaning up dep-selector-libgecode and the code duplication is a bit of codesmell.

i suspect this could be shipped generically as a cleanup routine that nuked all `*.a` files in native gems across all platforms (other than aix) and all gems, and just ran at the end of every build...